docs/embedded/concepts/app-concepts/containertypes.md

```diff
@@ -30,33 +32,19 @@ New-SPOContainerType
-
-> [!NOTE]
-> The Trial Container Type is the only Container Type that is restricted to the Partner (creating) tenant and cannot be shared with other consuming tenants.
->
-> It is not possible to convert a Trial Container Type and/or the associated Containers to a Standard Container Type.
-
-## Standard Container Types
-
-Standard Container Types are associated with a billing profile in [Azure portal](https://portal.azure.com) when they're created. The billing profile includes an Azure Subscription ID and a Region Group. There are two prerequisites to creating s Standard Container:
+The following restrictions are applied to Container Type in the Trial status:
+1. Up to five active Container instances of the Container Type can be created.
+2. Each Container instance has up to 1GB of storage space.
+3. The Container Type expires after 30 days and access to any existing Container instances of that type will be removed. 
+4. eveloper must delete all Container instances of an existing Container Type in Trial status to create a new Container Type for trial.
+5. The Container Type is restricted to work in the developer tenant. It cannot be deployed in other consuming tenants.
 
-- An Azure subscription and Resource Group must be present in the [Azure portal](https://portal.azure.com)
-- An App registration must be created in [Microsoft Entra ID](https://entra.microsoft.com)
+## Standard Container Type
 
-## Create a Standard ContainerType
-
-A developer admin can create a new billable Container Type using the following PowerShell cmdlet. Tenants can create a maximum of five Container Types. Trial Container Types don't count against the maximum number of Container Types that a Partner tenant can create.
-
-> [!IMPORTANT]
-> Assign the SharePoint Embedded Administrator role available in M365 Admin Center or Microsoft Entra to execute SharePoint Embedded Container commandlets mentioned in this article.
-> 
-> Global Administrators can continue to execute SharePoint Embedded container cmdlets.
-> 
-> If you are a SharePoint Administrator, grant yourself the SharePoint Embedded Admin role as well to execute these cmdlets.
-
-> [!NOTE]
-> The user or admin who will set up billing relationship for SharePoint Embedded will need to have owner or contributor permissions on the Azure subscription.
+Container Type can also be created with an Azure billing profile, which includes an Azure Subscription ID and a Region Group. This Container Type is in the Standard status and is billable. You need the following to create a Container Type in the Standard status:
+•	An Azure subscription and Resource Group must be present in the Azure portal
```

Comment:
Markdown unordered bullet points should use `-`, not the dot as used here.

Surround bullets with an empty line (IOW, there should be a blank line before line 45 & after line 46.